src/main/target/WORMFC/target.h

```diff
@@ -63,6 +66,8 @@
 #define INVERTER_PIN_UART3      PB12
 
 #define USE_VCP
+#define USB_MSC
```

Comment:
These seem to be unneeded.

Reply:
necessary for #5443, unless you wan't to do seperate commit out of it?

Reply:
They should be in #5443 then, for clarity.